cirq/ops/common_gates.py

```diff
@@ -1081,14 +1081,12 @@ class ISwapPowGate(eigen_gate.EigenGate,
     def _eigen_components(self):
         return [
             (0, np.diag([1, 0, 0, 1])),
-            (+0.5, np.array([[0, 0, 0, 0],
-                             [0, 0.5, 0.5, 0],
-                             [0, 0.5, 0.5, 0],
-                             [0, 0, 0, 0]])),
-            (-0.5, np.array([[0, 0, 0, 0],
-                             [0, 0.5, -0.5, 0],
-                             [0, -0.5, 0.5, 0],
-                             [0, 0, 0, 0]])),
+            (-0.5,
```

Comment:
Make the arrays more readable by calling `#yapf disable` around these parts.

Reply:
Done.

Reply:
I think also you can add a trailing ","?  
`[[0, 0, 0, 0], 
 [0, 0.5, 0.5, 0],
 [0, 0.5, 0.5, 0],
 [0, 0.5, 0.5, 0],
 [0, 0, 0, 0],]`